spark/interpreter/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -252,20 +253,28 @@ public SparkVersion getSparkVersion() {
     return sparkVersion;
   }
 
-  private String extractScalaVersion() throws InterpreterException {
-    String scalaVersionString = scala.util.Properties.versionString();
+  private String extractScalaVersion(SparkConf conf) throws InterpreterException {
+    String scalaVersionString;
+    if (conf.contains("zeppelin.spark.scala.version")) {
```

Comment:
It's better to add comment on when `zeppelin.spark.scala.version` is used and when `scala.util.Properties.versionString()` is used

Reply:
Thanks for the review @zjffdu.  I've added comment.